custom_event_checks/opslevel/service_infrastructure_check/aws-postgres-extended-support-check.yml

```
@@ -0,0 +1,34 @@
+---
+version: 1
+kind: generic
+spec:
+  name: AWS Postgres Extended Support Check
+  serviceSelector: ".[] | .name"
+  successCondition: '[.[] | select(.name == $ctx.alias) | .relatedResources.nodes[]
```

```suggestion
  successCondition: >-
    '[.[] | select(.name == $ctx.alias) | .relatedResources.nodes[]
```
The little `>-` is a [block_strip in these docs](https://learnxinyminutes.com/docs/yaml/). It takes the 3 lines below `successCondition` and combines them into one (when parsed), and removes any trailing newline characters.
Plus it makes things a little more readable ✨